onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -298,7 +311,13 @@ public void onInfoWindowClick(Marker marker) {
 
         // Show trip details screen for the vehicle associated with this marker
         ObaTripStatus status = mMarkerData.getStatusFromMarker(marker);
-        TripDetailsActivity.start(mActivity, status.getActiveTripId());
+
+        String focusedStopId = mController.getFocusedStopId();
```

Comment:
@azizmb9494 It looks like the Amazon implementation is slightly different here than the Google implementation.  Sometimes this can happen if the Gradle copy script doesn't execute properly when you're building the Google variant.  Could you try re-building this project again using any of the Google build variants (oba, agencyx, agencyy) to re-generate the Amazon code, and then push that new version here?